src/NuGet.Core/NuGet.Build.Tasks.Console/NuGet.Build.Tasks.Console.csproj

```diff
@@ -0,0 +1,55 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFrameworks Condition=" '$(IsVsixBuild)' != 'true' ">$(TargetFrameworksExe)</TargetFrameworks>
+    <TargetFramework Condition=" '$(IsVsixBuild)' == 'true' ">$(NETFXTargetFramework)</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <Shipping>true</Shipping>
+    <PackProject>true</PackProject>
+    <SkipShared>true</SkipShared>
+    <Description>NuGet restore console application.</Description>
+    <!-- NU5100: Suppress warning about a DLL not in a lib folder since on .NET Core the EXE is technically a DLL -->
+    <NoWarn>$(NoWarn);CS1591;NU5100</NoWarn>
+    <XPLATProject>true</XPLATProject>
+    <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
Did we run this by the GC team?
Curious what their response was?

Reply:
I have not had time to ask around.  I'm confident this is the correct choice.

Reply:
I think it's ok as well, just curious if they had an opinion :)

Reply:
Well, an opinion more than the readily available docs.